realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -635,6 +637,12 @@ void BaseRealSenseNode::frame_callback(rs2::frame frame)
                         publishFrame(f, t, COLOR, _depth_aligned_image, _depth_aligned_info_publisher, _depth_aligned_image_publishers, false);
                         continue;
                     }
+                    rs2::video_frame original_infra2_frame = frameset.get_infrared_frame(2);
```

Comment:
What will `get_infrared_frame(2)` return if we didn't turn on the ir?
Why not moving this into the if only when the user ask to align the depth?

Reply:
It will return a nullptr object.
I can move it in the if statement. I did it this way for code uniformity, as this is the way it is done in the color's if statement above.